src/test/run-make-fulldeps/coverage/coverage_tools.mk

```diff
@@ -14,10 +14,3 @@
 # Therefore, `-C link-dead-code` is no longer automatically enabled.
 
 UNAME = $(shell uname)
-
-# Rust option `-Z instrument-coverage` uses LLVM Coverage Mapping Format version 4,
```

Comment:
Presumably this file is not adding much value anymore and we should move the uname to tools.mk?

Reply:
Yeah but lets maybe leave that for a later PR. Or at least, this may need to be backported when I add the other beta backports that depend on it (maybe)?

Reply:
Hmm on second thought I  guess the backport isn't actually necessary since all it will mean is that the tests on beta will continue to be silently ignored